include/common.hpp

```diff
@@ -12,10 +12,11 @@ using eosio::symbol;
 namespace common
 {
 
-    static constexpr symbol S_HYPHA("HYPHA", 2);
-    static constexpr symbol S_HVOICE("HVOICE", 2);
+    static constexpr symbol S_REWARD("BM", 2);
+    static constexpr symbol S_VOICE("BMV", 2);
+    static constexpr symbol S_PEG("BUSD", 2);
+
```

Comment:
Changing the constants for Hypha specific tokens to REWARD, VOICE, and PEG tokens.

Reply:
TODO: Store this on the DHO settings doc as well